lisp/ess-custom.el

```diff
@@ -2108,27 +2113,26 @@ for help files.  The default value is nil for other systems."
 
 ;;;;; names for communication using MS-Windows 9x/NT ddeclient mechanism
 
-(defcustom inferior-ess-ddeclient nil
+(defcustom inferior-ess-ddeclient "Initial"
```

Comment:
> My plan is to revert the older 8-Nov-2010 version of the file into the current ESS and then make it work. 

That's too old. The 6-cleanup happened quite some years ago. If no-one complained for missing r-gui support since then, it probably means that no-one is using it. 

I don't see any point in keeping DDE support for R, R-s gui is rudimentary. Keeping it for S-plus doesn't make much sense either. There are fewer and fewer users of S-plus and those are unlikely to be ESS users anyways. ESS users are unlikely to use GUI either. So keeping it alive is a wasted effort IMO. 

That being said, if S-PLUS gui part could be isolated of the rest and sealed as you propose it would be good. We can put it in old/ box and forget about it.